sql/item_latents.sql

```diff
@@ -686,18 +682,22 @@ INSERT INTO `item_latents` VALUES(16238, 369, 1, 13, 3);
 INSERT INTO `item_latents` VALUES(14954, 5, 35, 14, 0);
 INSERT INTO `item_latents` VALUES(14954, 168, -5, 14, 0);
 INSERT INTO `item_latents` VALUES(14954, 71, 1, 14, 0);
-INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 2);
+
+-- -------------------------------------------------------
+-- Sirius Axe
+-- -------------------------------------------------------
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 3);
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 4);
 INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 5);
+INSERT INTO `item_latents` VALUES(17952, 366, 2, 16, 6);
 
 -- -------------------------------------------------------
 -- Company Sword
 -- -------------------------------------------------------
-INSERT INTO `item_latents` VALUES(17662, 366, 2, 15, 2); -- DMG 46~54, increases by 2 for every person in party (excl. wearer)
```

Comment:
Looking at every edits made in FFXIclopedia, the consensus seems to be that it starts at 3 characters in the party but it's really all over the place. 

http://ffxiclopedia.wikia.com/wiki/Company_Sword?oldid=195396
Company Sword has a hidden effect, activated when in a party of 3 or more. For every member in the party after the second, add 2 to the Base Damage portion of the damage calculation. 

http://ffxiclopedia.wikia.com/wiki/Company_Sword?oldid=354561
Damage on Company Sword increases by 2 for each party member over 2, up to 51 for a party of 6 people. 

http://ffxiclopedia.wikia.com/wiki/Company_Sword?oldid=389024
Damage on Company Sword increases by 1 for each party member over 2, up to 56 for a party of 6 people. 

http://ffxiclopedia.wikia.com/wiki/Company_Sword?oldid=461658
Damage on Company Sword increases by 2 for each party member over 2, up to 54 for a party of 6 people.